lib/email_api/email_template_schemas.yaml

```diff
@@ -47,6 +47,13 @@ shutoff_vm:
   html_filepath: "html/shutoff_vm.html.j2"
   plaintext_filepath: "plaintext/shutoff_vm.txt.j2"
 
+hypervisor_vm:
```

Comment:
This is fine - but maybe `vms_on_hypervisor`?
The other schemas might need changing names to be more informative as we add more